tests/src/lib.rs

```diff
@@ -341,6 +341,27 @@ mod tests {
         };
     }
 
+    #[test]
+    fn test_deep_nesting() {
+        fn build_and_roundtrip(depth: usize) -> Result<(), prost::DecodeError> {
```

Comment:
great test, but could you also add a case which covers the other nesting types, map, oneof, message and group?  This could be separate cases for each, or one case which nests those types within each other.

Reply:
and `repeated`.